api/tests/routes/backoffice/offerers_test.py

```diff
@@ -91,3 +332,387 @@ def test_cannot_get_offerer_users_as_anonymous(self, client):
+        offerer,
+    ):
+        # given
+        admin = users_factories.UserFactory()
+        auth_token = generate_token(admin, [Permissions.READ_PRO_ENTITY])
+
+        # when
+        response = client.with_explicit_token(auth_token).get(
+            url_for("backoffice_blueprint.get_offerer_total_revenue", offerer_id=offerer.id)
+        )
+
+        # then
+        assert response.status_code == 200
+        assert response.json["data"] == 0
+
+    @override_features(ENABLE_BACKOFFICE_API=True)
+    def test_offerer_returns_0_if_offerer_is_not_found(self, client):
+        # given
+        admin = users_factories.UserFactory()
+        auth_token = generate_token(admin, [Permissions.READ_PRO_ENTITY])
+
+        # when
+        response = client.with_explicit_token(auth_token).get(
+            url_for("backoffice_blueprint.get_offerer_total_revenue", offerer_id=42)
+        )
+
+        # then
+        assert response.status_code == 200
+        assert response.json["data"] == 0
+
+    @override_features(ENABLE_BACKOFFICE_API=True)
+    def test_cannot_get_offerer_without_permission(self, client):
+        # given
+        user = users_factories.UserFactory()
+        auth_token = generate_token(user, [])
+
+        # when
+        response = client.with_explicit_token(auth_token).get(
+            url_for("backoffice_blueprint.get_offerer_total_revenue", offerer_id=42)
+        )
+
+        # then
+        assert response.status_code == 403
+
+    @override_features(ENABLE_BACKOFFICE_API=True)
+    def test_cannot_get_offerer_as_anonymous(self, client, offerer):
+        # given
+        auth_token = generate_token(users_factories.UserFactory.build(), [Permissions.READ_PRO_ENTITY])
+
+        # when
+        response = client.with_explicit_token(auth_token).get(
+            url_for("backoffice_blueprint.get_offerer_total_revenue", offerer_id=offerer.id)
+        )
+
+        # then
+        assert response.status_code == 403
+
+
+class GetOffererOffersStatsTest:
+    @override_features(ENABLE_BACKOFFICE_API=True)
+    def test_offerer_offers_stats(
+        self,
+        client,
+        offerer,
+        offerer_active_individual_offers,
+        offerer_inactive_individual_offers,
+        offerer_active_collective_offers,
+        offerer_inactive_collective_offers,
+    ):
+        # given
+        admin = users_factories.UserFactory()
+        auth_token = generate_token(admin, [Permissions.READ_PRO_ENTITY])
+
+        # when
+        response = client.with_explicit_token(auth_token).get(
+            url_for("backoffice_blueprint.get_offerer_offers_stats", offerer_id=offerer.id)
+        )
+
+        # then
+        assert response.status_code == 200
+        offer_stats = response.json["data"]
+        assert offer_stats["active"]["individual"] == 5
+        assert offer_stats["active"]["collective"] == 5
+        assert offer_stats["inactive"]["individual"] == 5
+        assert offer_stats["inactive"]["collective"] == 5
```

Comment:
Est-ce judicieux d'en avoir 5 dans chacun des cas ? Ça ne permet pas de détecter d'erreur si on s'embrouille dans le code en intervertissant des requêtes.
Une suggestion : modifier les `for _ in range(5)` plus haut pour avoir 4 nombres différents.